src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient.swift

```diff
@@ -668,6 +668,7 @@ private extension URLRequest {
 		let submPayload = SAP_Internal_SubmissionPayload.with {
 			$0.requestPadding = self.getSubmissionPadding(for: payload.exposureKeys)
 			$0.keys = payload.exposureKeys
+			$0.checkIns = payload.eventCheckIns
 			/// Consent needs always set to be true https://jira.itc.sap.com/browse/EXPOSUREAPP-3125?focusedCommentId=1022122&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1022122
```

Comment:
The link to JIRA doesn't work, it should probably be removed.

Reply:
Removed dead link.